### ISSUE TITLE
Cultivated Soil incompatibility with Foraging and Farming mods

### ISSUE THREAD

**Issue by ma11088** [zolantris/ValheimMods#58]
### What happened?

This is something that has been a problem for a long time, just requesting that this feature can work with the foraging and farming mods.

### Log outputs

```txt
none
```

### Environment

```markdown
OS:
- **OS**: Windows 11 | Ubuntu 20.04
- **Graphics**: DX11 (or Vulkan)
Mods:
- **BepInEx**: 5.4.2202
- Jotunn 2.15.2
- **ValheimRAFT**: Major.Minor.Patch
```

**Comment by zolantris** [comment]
Yes I can look into it. It'll probably be a month, working on v2.0.0 features first.

**Comment by Felousia** [comment]
this would be lovely to have. For a while I wasn't sure why my soil tiles didn't work at all but just as I'm about to right an issue ticket i thought ehhhh maybe i should look first. So at least now i know why it doesn't work. It was the farming skill mod from smoothbrain it is incompatible with it seems. disabled that and it worked out. i look forward to seeing what you are working on now and future compatibility with the farming mods ^^. but for now, seems my main land base will still have use aside from being close to the spawn.
